.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
+  - 7.3
```

Comment:
Why are we building this against 7.3 instead of 7.2? Future proofing?

Reply:
Tests don’t even work right now and haven’t for a while. Was my attempt at making them work as the errors thrown were because of PHP 5.4.